package.json

```diff
@@ -2,15 +2,26 @@
   "name": "2024-jan-fst-bank-app-backend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "dist/app.js",
   "scripts": {
```

Comment:
You could add a "start" script here to run `node dist/app.js` when you type `npm start`